pokemongo_bot/cell_workers/incubate_eggs.py

```diff
@@ -57,6 +60,17 @@ def _apply_incubators(self):
             for egg in self.eggs:
                 if egg["used"] or egg["km"] == -1:
                     continue
+                
+                if self.breakable_incubator:
+                    if incubator.get('uses_remaining') is not None:
+                        if egg["km"] not in self.breakable_incubator:
+                            continue
+                    
+                if self.infinite_incubator:
+                    if incubator.get('uses_remaining') is None: 
```

Comment:
No worries. I added the lines of comment just before your message, I agree that the line is a little obscure. :)